docs/docs_source/build_test_ship/shipping.rst

```diff
@@ -0,0 +1,146 @@
+
+..  _go live:
+..  _shipping:
+
+Shipping your Office Online integration
+=======================================
+
+..  spelling::
+
+    rollout
+    Rollout
+
+Once you believe you are ready to ship your integration, you should contact Microsoft and explain that you believe
+you are ready to ship. This will begin the 'go live' process, after which you will be able to ship your Office Online
+integration.
+
+The 'go live' process consists of three phases:
+
+#. **Validation** - Microsoft will validate your WOPI implementation as well as your UI integration. Depending on what
+   issues are uncovered during validation, this may take some time. You should estimate a two-week turnaround time
+   assuming there are no major issues uncovered. However, we recommend allowing at least a month unless your
+   integration is very simple.
+
+   ..  tip::
+       You can avoid delays in validation by ensuring that your implementation is consistent with this documentation
+       and that the :ref:`validator` tests are passing.
+
+#. **Production Smoke Test** - Microsoft will enable you to use the production Office Online environment for
+   smoke-testing. This should be basic testing performed by the host since the production environment is slightly
+   different than the test environment, and different issues may be uncovered. This can be as long or as short as the
+   host deems necessary. Typically a week is enough time, though it can be much shorter.
+
+#. **Sign off and roll out** - Once Microsoft has signed off on your integration, you can begin to roll out to your
+   users. Depending on your traffic estimates, Microsoft may request that you roll out over a period of several days
+   to ensure you do not overload Office Online.
```

Comment:
instead of "overload Office Online" suggest "overload either Office Online or your WOPI servers."